packages/e2e/test/environment.ts

```diff
@@ -1,29 +1,84 @@
 import * as envalid from 'envalid';
 
-export const getEnv = (override: NodeJS.ProcessEnv = {}) =>
+/**
+ * Shared collection of all the environment configuration variables with their constraints
+ */
+const variablesConfiguration = {
```

Comment:
```suggestion
const validators = {
```
Let's stick with the terminology `envalid` uses